buildSrc/src/main/java/org/opensearch/gradle/testfixtures/TestFixturesPlugin.java

```diff
@@ -112,27 +112,42 @@ public void apply(Project project) {
             project.getPluginManager().apply(BasePlugin.class);
             project.getPluginManager().apply(DockerComposePlugin.class);
 
-            TaskProvider<Task> preProcessFixture = project.getTasks().register("preProcessFixture", t -> {
-                t.getOutputs().dir(testfixturesDir);
-                t.doFirst(t2 -> {
-                    try {
-                        Files.createDirectories(testfixturesDir.toPath());
-                    } catch (IOException e) {
-                        throw new UncheckedIOException(e);
-                    }
-                });
+            TaskProvider<Task> preProcessFixture = project.getTasks().register("preProcessFixture", new Action<Task>() {
```

Comment:
Reason: Using Java lambdas is not supported as task inputs. Please refer to https://docs.gradle.org/7.3/userguide/validation_problems.html#implementation_unknown for more details about this problem.